client/src/common/date.ts

```diff
@@ -19,17 +19,17 @@ export function formatTimeSince(
   if (!diff || (diff.years && diff.years > 0) || (diff.months && diff.months > 0) || (diff.days && diff.days > 6)) {
     return formatDate(date, format);
   } else if (roundDays) {
-    return translate('common.date.lastLoginDays', { days: diff.days || 0 }, diff.days || 0);
+    return msTranslate({ key: 'common.date.lastLoginDays', data: { days: diff.days || 0 }, count: diff.days || 0 });
```

Comment:
All the functions in `common/` should not call `msTranslate()`, but instead return a `Translatable`.